s3tests_boto3/functional/test_s3.py

```diff
@@ -15131,6 +15140,83 @@ def _bucket_logging_key_filter(log_type):
         assert _verify_records(body, src_bucket_name, 'REST.PUT.OBJECT', names, log_type, expected_count, exact_match=True)
 
 
+def _bucket_logging_acl_required():
+    src_bucket_name = get_new_bucket_name()
+    src_bucket = get_new_bucket_resource(name=src_bucket_name)
+    log_bucket_name = get_new_bucket_name()
+    log_bucket = get_new_bucket_resource(name=log_bucket_name)
+    client = get_client()
+    prefix = 'log/'
+    key = 'my-test-object'
+    _set_log_bucket_policy(client, log_bucket_name, [src_bucket_name], [prefix])
+
+    #client.put_bucket_acl(Bucket=src_bucket_name, ACL='public-read')
```

Comment:
any reason why not ACL at bucket level?

Reply:
The Read ACL at the bucket level allows the grantee to list the objects in the bucket. To actually download the object, the ACL is required to be set at the object level.